projects/demo-integrations/cypress/tests/desktop/kit/input/input.cy.ts

```diff
@@ -112,4 +111,13 @@ describe(`Input`, () => {
         cy.get(`@input`).scrollTo(`center`, {ensureScrollable: false});
         cy.get(`@wrapper`).matchImageSnapshot(`08-horizontally-scrolled`);
     });
+
+    it(`external mask works`, () => {
+        cy.tuiVisit(`${INPUT_PAGE_URL}`);
+
+        cy.get(`tui-doc-example[id=mask]`).tuiScrollIntoView().as(`wrapper`);
+        cy.get(`@example`).find(`tui-input input[tuiTextfield]`).as(`input`);
```

Comment:
wrapper

Reply:
<img width="1000" alt="image" src="https://user-images.githubusercontent.com/12021443/203837228-7aaf1b1a-9afa-4b61-8380-d3cebb788421.png">

Reply:
Someday I'll get it right :D